bump_rpm.sh

```diff
@@ -48,13 +48,14 @@ if [[ $CURRENT_VERSION != $NEW_VERSION ]] ; then
 	echo "${PACKAGE_NAME}: $CURRENT_VERSION != $NEW_VERSION ; bumping"
 	sed -i "s/^\(Version:\s\+\).\+$/\1${NEW_VERSION}/" $SPEC_FILE
 
+	EPOCH=$(rpmspec --srpm -q --queryformat='%{evr}' --undefine=dist $SPEC_FILE | grep -o '^.*:' || :)
```

Comment:
A better solution is to do this after resetting the release. Then the `%{evr}` is exactly the format and you can drop the grep part.

Reply:
Very good idea, updated and looks much better now. Thanks for the suggestion.